frontend/src/components/OdhAppCard.tsx

```diff
@@ -53,23 +59,11 @@ const OdhAppCard: React.FC<OdhAppCardProps> = ({ odhApp }) => {
       View documentation
       <ExternalLinkAltIcon />
     </DropdownItem>,
+    <DropdownItem key="quick-start" className={quickStartClasses} href="#" onClick={onQuickStart}>
+      {`${getLaunchStatus(odhApp.spec.quickStart || '', qsContext)} quick start`}
```

Comment:
Should we not add the quick start action if there is no `odhApp.spec.quickStart`?